libgstc/libgstc.c

```diff
@@ -392,6 +392,24 @@ gstc_pipeline_stop (GstClient * client, const char *pipeline_name)
   return gstc_cmd_change_state (client, pipeline_name, state);
 }
 
+GstcStatus
+gstc_pipeline_get_graph (GstClient * client, const char *pipeline_name, char **response)
+{
+
+  GstcStatus ret;
+
+  char uri[sizeof("/pipelines/%s/graph") + sizeof(pipeline_name)];
```

Comment:
This has several errors:
1. `sizeof(pipeline_name)`: returns the size of a pointer (4 or 8), not the size of the string.
2. You are missing 1 byte for the trailing null character.
3. The rest of the code is following one pattern, why aren't you following it?